source/docs/yearly-overview/removed-features.rst

```diff
@@ -0,0 +1,69 @@
+# Removed features
+
+## Labview
+NI will not support Labview on SystemCore.
+
+## Java tools
+
+All Java GUI tools have been removed due to lack of maintenance.
+
+### Shuffleboard
+Use Elastic as a driver dashboard, use AdvantageScope for debugging and data visualization.
+
+### SmartDashboard
+Use Elastic as a driver dashboard, use AdvantageScope for debugging and data visualization.
```

Comment:
I'd also say that it's removed because it only supports NetworkTables 3 and doesn't support modern platforms.